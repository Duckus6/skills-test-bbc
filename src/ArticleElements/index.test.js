import React from 'react';
import {mockParagraph,mockHeading,mockImage,mockListUnordered,mockListOrdered} from "../fixtures/elements";
import Paragraph from "./paragraph";
import Heading from "./heading";
import List from "./list";
import Image from "./image";







describe("the server returns 200 code",()=> {
    test("renders paragraph element",()=>{
        expect(Paragraph(mockParagraph)).toMatchSnapshot()
    });
    test("renders heading element",()=>{
        expect(Heading(mockHeading)).toMatchSnapshot()
    });
    test("renders image element",()=>{
        expect(Image(mockImage)).toMatchSnapshot()
    });
    describe("render lists",() =>{
        test("renders unordered list",()=>{
           expect(List(mockListUnordered)).toMatchSnapshot()
        });
        test("renders ordered list",()=>{
           expect(List(mockListOrdered)).toMatchSnapshot()
        });
    });
});