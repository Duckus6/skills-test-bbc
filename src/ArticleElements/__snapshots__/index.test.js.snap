// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the server returns 200 code render lists renders ordered list 1`] = `
<ol>
  <li>
    item1
  </li>
  <li>
    item2
  </li>
</ol>
`;

exports[`the server returns 200 code render lists renders unordered list 1`] = `
<ul>
  <li>
    itemA
  </li>
  <li>
    itemB
  </li>
</ul>
`;

exports[`the server returns 200 code renders heading element 1`] = `
<h4>
  mockHeading
</h4>
`;

exports[`the server returns 200 code renders image element 1`] = `
<img
  alt="image text"
  height="420"
  src="https://avatars1.githubusercontent.com/u/16746643?s=460&v=4"
  width="640"
/>
`;

exports[`the server returns 200 code renders paragraph element 1`] = `<p />`;
